init/fluent_bit_init_process_test.go

```diff
@@ -94,18 +97,20 @@ func TestSetECSTaskMetadata(t *testing.T) {
 
 	// Test case 1: full metadata
 	metadataTest1 := ECSTaskMetadata{
-		AWS_REGION:          "us-west-2",
-		ECS_CLUSTER:         "ecs-Test",
-		ECS_TASK_ARN:        "arn:aws:ecs:us-west-2:111:task/ecs-local-cluster/37e8",
-		ECS_TASK_ID:         "56461",
-		ECS_FAMILY:          "esc-task-definition",
-		ECS_LAUNCH_TYPE:     "EC2",
-		ECS_REVISION:        "1",
-		ECS_TASK_DEFINITION: "esc-task-definition:1",
+		AWS_REGION:            "us-west-2",
+		AWS_AVAILABILITY_ZONE: "us-west-2a",
+		ECS_CLUSTER:           "ecs-Test",
+		ECS_TASK_ARN:          "arn:aws:ecs:us-west-2:111:task/ecs-local-cluster/37e8",
+		ECS_TASK_ID:           "56461",
+		ECS_FAMILY:            "esc-task-definition",
+		ECS_LAUNCH_TYPE:       "EC2",
+		ECS_REVISION:          "1",
+		ECS_TASK_DEFINITION:   "esc-task-definition:1",
 	}
 
-	expectedContent1 := "export FLB_AWS_USER_AGENT=init\n" +
+	expectedContent1 := "export FLB_AWS_USER_AGENT=ecs-init\n" +
```

Comment:
The default for this value appears to have changed in db3fd7f9659151d629e93107818c3b8dbe9ddd0f.